cypress/support/generateData.js

```diff
@@ -0,0 +1,19 @@
+const { faker } = require('@faker-js/faker');
+
+export const generateCheckoutData = () => {
+  const name = faker.name.fullName();
+  const country = faker.address.country();
+  const city = faker.address.city();
+  const creditCard = faker.finance.creditCardNumber();
+  const month = faker.date.month();
+  const year = faker.date.future().getFullYear();
```

Comment:
The use of `faker.date.future().getFullYear()` for generating a year might result in a year that is too far in the future for a credit card expiration date. Consider using a more appropriate method to generate a realistic expiration year.